frontend/lib/features/audio-player/audio-player-screen.widget.dart

```diff
@@ -0,0 +1,207 @@
+    _player = AudioPlayer();
+
+    _player.setAudioSource(
+        AudioSource.uri(
+          Uri.parse(
+            'https://s3.amazonaws.com/scifri-episodes/scifri20181123-episode.mp3'
+          )
+        )
+      );
+
+    
+  }
+
+  @override
+  void dispose() {
+    _player.dispose();
+    super.dispose();
+  }
+
+  void didChangeAppLifecycleState(AppLifecycleState state) {
+    if (state == AppLifecycleState.paused) {
+      _player.stop();
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: StandardColor.primary,
+      appBar: AppBar(
+        backgroundColor: Colors.transparent,
+        elevation: 0,
+        leadingWidth: 90.0,
+        leading: Padding(
+          padding: const EdgeInsets.symmetric(horizontal: 10.0),
+          child: InkWell(
+            onTap: () {
+              Navigator.pop(context);
+            },
+            child: Row(
+              children: [
+                const Icon(Icons.arrow_back_ios, color: Colors.black, size: 16.0,),
+                const SizedBox(width: 3.0,),
+                Text('Back', style: StandardText.button,)
+              ],
+            ),
+          ),
+        ),
+      ),
+      body: Column(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        mainAxisSize: MainAxisSize.max,
+        mainAxisAlignment: MainAxisAlignment.end,
+        children: [
+          Container(
+            alignment: Alignment.bottomCenter,
+            decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(20),
+               color: StandardColor.textContrastColor),
+            child: Padding(
+              padding: const EdgeInsets.symmetric(vertical: 18.73),
+              child: Column(
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.symmetric(horizontal: 20.0),
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children:[
+                            Text(widget.title, style: StandardText.headline6,),
+                            const SizedBox(height: 5,),
+                            Text(widget.speaker, style: StandardText.subtitle2,),
+                          ],
+                        ),
+                        const Icon(Icons.favorite_border_outlined) // Nur zum Testzweck
+                      ],
+                    ),
+                  ),
+                  const SizedBox(height: 19.0,),
+                  CustomProgressBar(positionDataStream: _positionDataStream, player: _player),
+                  const SizedBox(height: 44.0,),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    children: [
+                      IconButton(
```

Comment:
Seek Button code is duplicated. Only difference seems to be the + or - 10 seconds. Create a widget that can handle both cases.